ansible/roles/baselayout/vars/main.yml

```diff
@@ -57,27 +57,27 @@ packages: {
   ],
 
   debian7: [
-    'gcc-4.8,g++-4.8,sudo',
+    'gcc-4.8,g++-4.8,sudo,libcap-dev',
```

Comment:
AFAIK we do not have debian7 in the CI anymore and we can get rid of the `debian7` section (perhaps in a separate PR/commit).

Reply:
I've opened #2608 for this.